src/libraw.mk

```diff
@@ -0,0 +1,31 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := libraw
+$(PKG)_WEBSITE  := https://libraw.org
+$(PKG)_DESCR    := A library for reading RAW files obtained from digital photo cameras
+$(PKG)_VERSION  := 0.19.5
+$(PKG)_CHECKSUM := 9a2a40418e4fb0ab908f6d384ff6f9075f4431f8e3d79a0e44e5a6ea9e75abdc
+$(PKG)_GH_CONF  := LibRaw/LibRaw/releases
+$(PKG)_DEPS     := cc jpeg lcms
+
+define $(PKG)_UPDATE
+    echo 'TODO: write update script for $(PKG).' >&2;
+    echo $($(PKG)_VERSION)
+endef
```

Comment:
The _UPDATE section can be removed, it's set by GH_CONF. You can test it with `make update-package-libraw`